client/gutenberg/extensions/slideshow/style.scss

```diff
@@ -8,21 +8,19 @@
 		height: 400px; // This is a default, which will be replaced programmatically
 		overflow: hidden;
 		opacity: 0;
+
 		&.wp-swiper-initialized {
 			opacity: 1;
 		}
 	}
 
 	.wp-block-jetpack-slideshow_slide {
-		margin: 0;
-	}
-
-	.wp-block-jetpack-slideshow_image-container {
 		align-items: center;
-		background: #f0f0f0;
+		background: rgba( 0, 0, 0, 0.1 );
```

Comment:
Why not just make this `background: transparent`?

Reply:
To have a very subtle background that determines the boundaries of the slideshow and make it a complete block. A transparent background would make the arrows almost "floating" if the image wasn't wide enough.